pallets/crowdloans/src/lib.rs

```diff
@@ -453,7 +494,7 @@ pub mod pallet {
                 &amount,
             );
 
-            Self::deposit_event(Event::<T>::VaultContributing(
+            Self::deposit_event(Event::<T>::VaultDoContributing(
```

Comment:
shall this event be deposited in `do_contribute`?

Reply:
yes，the same as do_withdraw